md/release-notes.md

```diff
@@ -1,18 +1,143 @@
+
+<a id="rest-timeout"/>
+
+### REST connector Read timeout is configurable
+In the REST connector, it is now possible to set the connection timeout (during connection) and the socket timeout (lack of answer of the distant machine after connection).
+The default value is one minute for both, the values that were hard-coded before.
+
+<a id="ldap-error"/>
+
+### LDAP synchronizer manages errors and continues on fail
+Better error management during LDAP synchronization.
+
+<a id="clause-in"/>
+
+### BDM query parameters accept arrays
+It is now possible to query for multiple values of an attribute in the BDM objects.
+
+<a id="translate-expression"/>
+
+### In the UI Designer, strings in JavaScript expressions can now be translated
+When writing the content of a variable of type JavaScript Expression, autocompletion now offers a new service for translation; uiTranslate().
+
+<a id="metadata"/>
+
+### Add a display name and display description to a page, form, or layout from the Editor
+Display name and description are used when the page, form, or layout are imported in the portal's Resources page.
+They could be edited by opening the page.properties file. Now, they can be directly edited in the page, form, or layout editor of the UI Designer, by clicking on the new "Tag" button.
+
+<a id="bonita-calls"/>
+
+### Get a summary of the calls to Bonita REST API made in a page, form, or layout
+Also by clicking on the "Tag" button, you can get a summary of all REST API calls made by the current artifact to the Bonita REST API.
+This helps validating if all expected calls are well implemented in the artifact in its current state.
+
+<a id="bonita-theme"/>
+
+### New Living Applications theme
+A new theme is available for any Living Application you create: "bonita-default-theme". 
+
+<a id="operations-apiaccessor"/>
+
+### Warning on best practices when using API in operations
+In diagram operations, the right operand should only use the apiAccessor with read-only methods. After helping several customers with scripts calling the apiAccessor with "write"-type methods, we have created a warning in the studio, when detecting the apiAccessor is called in an opreation script.
 
 <a id="technology-updates"/>
 
 ## Technology updates
 
+<a id="kerberos"/>
+
+### Integration of SSO Kerberos (Teamwork, Efficiency, Performance edition)
+Bonita works with Single-Sign-On (SSO) solutions using Kerberos protocol:
+- Use standard authentication service
+- Connect to Bonita Living Applications with company credentials (per tenant configuration)
+- Works for Windows and many UNIX and UNIX-like operating systems: FreeBSD, Apple's MacOS, RED Hat Enterprise Linux, Oracle's Solaris..
```

Comment:
I wouldn't specify the other operating system since we did not test it on them. Some people report they made it work with solaris or some linux OS on the Spnego filter forum but we did not test it with our integration.
Maybe, "compatible with Windows authentication and many UNIX-like operating systems"